src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -21,19 +24,68 @@ const EditShiftHours = ({ setShowEditShiftHours }) => {
     setSummaryErrors(errorFields);
   };
 
-  return (
-    <div>
-      <form
-        onSubmit={handleSubmit((data) => {
+  const onSubmit = async (formData) => {
+    dayjs.extend(utc);
+
+    const actualStartDate = dayjs(formData['startDate']).format('YYYY-MM-DD');
+    const startTime = formData[`${inputName}-start-time`];
+    const actualStartDateTime = dayjs(actualStartDate + ' ' + startTime)
+      //.utc()
+      .format();
+
+    const endTime = formData[`${inputName}-finish-time`] || null;
+    let actualEndDateTime;
+    if (endTime) {
+      actualEndDateTime = dayjs(actualStartDate + ' ' + endTime)
+        //.utc()
+        .format();
+    }
+    const tempNow = dayjs(new Date()).format();
+    const timecardPayload = {
+      ownerId: 1,
+      timePeriodTypeId: formData['timePeriodTypeId'],
+      actualStartTime: actualStartDateTime,
+      actualEndTime: actualEndDateTime,
+      createdAt: tempNow,
+      updatedAt: tempNow,
+    };
+
+    try {
+      const params = new URLSearchParams([
+        ['tenantId', '00000000-0000-0000-0000-000000000000'],
```

Comment:
It is just my personal opinion, but I guess like we're going to pull tenantId from URL or headers in the near future. So maybe you could add a comment that this is a temporary 'mock' aka hardcoded value? Not sure if I am correct here but my OCD  asked me to comment on it. Correct me if I am wrong. Thanks, Holly.

Reply:
good shout, you are right it will come from 'somewhere else'. It's on my list of 'things to discuss' 
I did have a comment, but it's untidy enough so I removed it now we have **THE LIST** 
Until we have discussed a little further, I just going to leave it for now.